assembl/static2/js/app/components/debate/survey/post.jsx

```diff
@@ -51,31 +53,35 @@ class Post extends React.Component {
   }
   handleAddSentiment(target, type) {
     const { id } = this.props;
-    this.props.addSentiment({ variables: { postId: id, type: type } })
-    .then((sentiments) => {
-      target.setAttribute('class', 'sentiment sentiment-active');
-      this.setState({
-        like: sentiments.data.addSentiment.post.sentimentCounts.like,
-        disagree: sentiments.data.addSentiment.post.sentimentCounts.disagree,
-        mySentiment: sentiments.data.addSentiment.post.mySentiment
+    this.props
+      .addSentiment({ variables: { postId: id, type: type } })
+      .then((sentiments) => {
+        target.setAttribute('class', 'sentiment sentiment-active');
+        this.setState({
+          like: sentiments.data.addSentiment.post.sentimentCounts.like,
+          disagree: sentiments.data.addSentiment.post.sentimentCounts.disagree,
+          mySentiment: sentiments.data.addSentiment.post.mySentiment
+        });
+      })
+      .catch((error) => {
+        displayAlert('danger', `${error}`);
       });
-    }).catch((error) => {
-      displayAlert('danger', `${error}`);
-    });
   }
   handleDeleteSentiment(target) {
     const { id } = this.props;
-    this.props.deleteSentiment({ variables: { postId: id } })
-    .then((sentiments) => {
-      target.setAttribute('class', 'sentiment');
-      this.setState({
-        like: sentiments.data.deleteSentiment.post.sentimentCounts.like,
-        disagree: sentiments.data.deleteSentiment.post.sentimentCounts.disagree,
-        mySentiment: sentiments.data.deleteSentiment.post.mySentiment
+    this.props
+      .deleteSentiment({ variables: { postId: id } })
+      .then((sentiments) => {
+        target.setAttribute('class', 'sentiment');
+        this.setState({
+          like: sentiments.data.deleteSentiment.post.sentimentCounts.like,
+          disagree: sentiments.data.deleteSentiment.post.sentimentCounts.disagree,
+          mySentiment: sentiments.data.deleteSentiment.post.mySentiment
```

Comment:
Honestly, it's a bad idea to use the local state to store this data. I knew the problem and I should refactor this part. @cedricmessiant : Now that the admin section is done, I have a better understanding on how to do that. Get data with a graphQL query and put it in the store. Right?